sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -64,17 +74,41 @@ public String getPlatform() {
         return platform;
     }
 
+    public Map<String, Object> getVars() {
+        return vars;
+    }
+
     /**
      * Convert an array of {@link StackTraceElement}s to {@link SentryStackTraceElement}s.
      *
      * @param stackTraceElements Array of {@link StackTraceElement}s to convert.
      * @return Array of {@link SentryStackTraceElement}s.
      */
     public static SentryStackTraceElement[] fromStackTraceElements(StackTraceElement[] stackTraceElements) {
+        Frame[] localsCache = LocalsCache.getCache();
         SentryStackTraceElement[] sentryStackTraceElements = new SentryStackTraceElement[stackTraceElements.length];
+        boolean mayHaveLocals = localsCache.length == stackTraceElements.length;
```

Comment:
Oops, yeah, so we have no way to ensure the last exception thrown (and thus cached) is the one the user is sending:

```java
Exception e;

try {
  blowUp1();
} catch (Exception e1) {
  e = e1;
}

try {
  blowUp2();
} catch (Exception e2) {
  // pass
}

// sending the not-most-recent-exception, which we won't have locals cached for
Sentry.capture(e);
```

This gets to the `Method` thing below. I want to see about comparing the method names to the stacktrace. The length thing was a hack that I forgot about.